src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDao movieDao;
+
+    public MovieServiceImpl() {
+        movieDao = new MovieDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
Consider passing the `SessionFactory` as a parameter to the constructor of `MovieServiceImpl` instead of directly accessing `HibernateUtil.getSessionFactory()`. This approach improves testability and decouples the service from the utility class.